editor/block-settings-menu/content.js

```diff
@@ -34,7 +35,7 @@ function BlockSettingsMenuContent( { mode, uids, isSidebarOpened, onDelete, onTo
 			>
 				{ __( 'Settings' ) }
 			</IconButton>
-			{ count === 1 &&
+			{ count === 1 && blockType && blockType.supportHTML !== false &&
```

Comment:
Is `count === 1` redundant here, since `blockType` assumes a single selected block anyways?